api/jobs.yaml

```diff
@@ -573,13 +573,25 @@ definitions:
     required:
       - failure
     properties:
+      taskName:
+        type: string
+        description: The name of the task that generated the failure message
```

Comment:
This should specify "if any" - This object is referenced both on the task and on the job, and dsub doesn't have tasks.

Reply:
done